src/routes/admin/profiles/amtProfileValidator.ts

```diff
@@ -11,6 +11,7 @@ export const amtProfileValidator = (): any => {
     check('profileName')
       .not()
       .isEmpty()
+      .toLowerCase()
```

Comment:
I think as long as the comparison during _activation_ is toLowered, we don't need to toLower() their input right? We can leave it alone?

Reply:
While we get it from Database it would be a problem as it is case sensitive.